algoliasearch-core/src/test/java/com/algolia/search/integration/client/MultiClusterManagementTest.java

```diff
@@ -48,16 +49,32 @@ void mcmTest() throws ExecutionException, InterruptedException {
 
     removeUserId(mcmClient, userID);
 
-    ListUserIdsResponse listUserIds2 = mcmClient.listUserIDsAsync().get();
+    removePastUserIDs(mcmClient);
+  }
+
+  void removePastUserIDs(SearchClient mcmClient) throws ExecutionException, InterruptedException {
+
+    int page = 0;
+    int hitsPerPage = 100;
+    List<UserId> userIDsToRemove = new ArrayList<>();
+
+    while (true) {
+      ListUserIdsResponse listUserIds = mcmClient.listUserIDsAsync(page, 100, null).get();
```

Comment:
Use `hitsPerPage` variable instead of hardcoded `100` value here.

Reply:
👍